<ui version="4.0" >
 <class>FolksonomyOptionsPage</class>
 <widget class="QWidget" name="FolksonomyOptionsPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>267</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="rename_files_3" >
     <property name="title" >
      <string>Folksonomy Tags</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QLabel" name="ignore_tags_2" >
        <property name="text" >
         <string>Ignore tags:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ignore_tags" />
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_5" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Minimal tag usage:</string>
          </property>
          <property name="buddy" >
           <cstring>min_tag_usage</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="min_tag_usage" >
          <property name="suffix" >
           <string> %</string>
          </property>
          <property name="maximum" >
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_6" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Maximal number of tags:</string>
          </property>
          <property name="buddy" >
           <cstring>min_tag_usage</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="max_tags" >
          <property name="maximum" >
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="ignore_tags_4" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
            <horstretch>4</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Join multiple tags with:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="join_tags" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editable" >
           <bool>true</bool>
          </property>
          <item>
           <property name="text" >
            <string/>
           </property>
          </item>
          <item>
           <property name="text" >
            <string> / </string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>, </string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>181</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
